### ISSUE TITLE
Tokenizer step is not understanding??

### ISSUE THREAD

**Issue by Iqra-Yousaf** [keras-team/keras#15658]
I have an issue related to tokenization, I have dataset of 5760 rows and predict multi-class labels. can any one elaborate me how to choose num_words and length of word index must be 5670. after tokenization and pad sequences , I use my pre-trained embedding of my dataset but in that vocabulary size decrease to 1192 by picking unique words in embedding. Then word_index length decrease i think and when we create embedding matrix for input in CNN model then length of  word_index what will be mentioned? I am newbie in python and i am confusing please elaborate where i am wrong. Thanks

**Comment by chunduriv** [comment]
@Iqra-Yousaf, Please refer [link1](https://neptune.ai/blog/tokenization-in-nlp), [link2](https://machinelearningknowledge.ai/keras-tokenizer-tutorial-with-examples-for-fit_on_texts-texts_to_sequences-texts_to_matrix-sequences_to_matrix/), [link3](https://stackoverflow.com/questions/61760508/how-to-choose-num-words-parameter-for-keras-tokenizer), [link4](https://stackoverflow.com/questions/51956000/what-does-keras-tokenizer-method-exactly-do) may help you regarding Tokenizer. This is not a bug or feature request, for any further queries you may open this issue in tf discussion [forum](https://discuss.tensorflow.org/) as there is a larger community there. Thanks!
